### PR TITLE
[FIX] l10n_pe, base_vat: Finish prepration for l10n_pe_edi

### PR DESCRIPTION
**Pull request by nhomar**
- base_vat: Correct management of the check of peruvian VAT without prefix.
- l10n_pe: Correct income account the last one is not correct.
- l10n_pe: Forced Round globally for peruvian companies once l10n_pe is
installed, and with the onchange.
- l10n_pe: For peruvian companies it does not make sense a sequence per
year and the year in the prefix is incorrect, we must force XXX- as
a sequence prefix.

### CODE REVIEW

File: addons/l10n_pe/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    @api.model
+    def _get_sequence_prefix(self, code, refund=False):
+        """For peruvian companies we can not use sequences with **/** due to the edi generation which need in the
+        sequence a plain text ended by a *-* and the length of this prefix"""
+        if self.env.company.country_id != self.env.ref('base.pe'):
+            return super()._get_sequence_prefix(code, refund=refund)
+        prefix = code.upper()
```
Comment (jco-odoo) [inline addons/l10n_pe/models/account_journal.py]: So, the standard shortcode is ok as a prefix?  And why do we remove one char for the refund?  I tested this in runbot and in my sequence I have no prefix at all.

Comment (nhomar) [reply addons/l10n_pe/models/account_journal.py]: We must have 3 chars followed by a dash (-), the removal is only referential for Credit note.

do this answer the question?

File: addons/l10n_pe/data/res_country_data.xml [hunk 2]
```diff
@@ -9,28 +9,30 @@
                 <div class="o_address_format">
                     <field name="parent_id" invisible="1"/>
                     <field name="type" invisible="1"/>
-                    <field name="street" invisible="1"/>
-                    <field name="street_name" placeholder="Street Name..." attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
-                    <field name="street2" placeholder="Street2"/>
+                    <field name="street_name" placeholder="Street Name..." attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}" class="oe_read_only"/>
+                    <field name="street" placeholder="Street" class="oe_edit_only"/>
+                    <field name="street2" placeholder="Street2" invisible="1"/>
```
Comment (jco-odoo) [inline addons/l10n_pe/data/res_country_data.xml]: So, no appartment?  People have to still use this for international addresses.

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**